recipes/lerclib/all/patches/add-CMakeLists-and-static.patch

```diff
@@ -0,0 +1,61 @@
+--- /dev/null
++++ b/CMakeLists.txt
+@@ -0,0 +1,24 @@
++cmake_minimum_required(VERSION 3.0.2)
++project(Lerc)
++
++file(GLOB SOURCES 
++  "src/LercLib/*"
++  "src/LercLib/Lerc1Decode/*"
++)
++
++add_library(LercLib ${SOURCES})
++set_property(TARGET LercLib PROPERTY CXX_STANDARD 11)
++if(BUILD_SHARED_LIBS)
```

Comment:
It would be nice if these things would be upstreamed.
The upstream cmake script has none of these options.
That way, we won't need patches in future releases.

Reply:
https://github.com/Esri/lerc/issues/125

Reply:
It doesn't look like upstream will answer soon.
If you're finished, ask for a final review so this can get merged.